docs/explanation/system-snaps/udisks2/report-a-bug.md

```diff
@@ -0,0 +1,10 @@
+(explanation-system-snaps-udisks2-reference-report-a-bug)=
+# Report a Bug
+
+Bugs can be reported [here](https://bugs.launchpad.net/snappy-hwe-snaps/+filebug).
```

Comment:
I'd rather point to github issues, we are not really using snappy-hwe-bugs anymore.